firmware/application/apps/ui_recon.cpp

```diff
@@ -719,9 +735,15 @@ namespace ui {
 		//HELPER: Pre-setting a manual range, based on stored frequency
 		rf::Frequency stored_freq = persistent_memory::tuned_frequency();
 		receiver_model.set_tuning_frequency( stored_freq );
-		frequency_range.min = stored_freq - 1000000;
+		if( stored_freq - 1000000 > 0 )
+			frequency_range.min = stored_freq - 1000000;
+		else
+			frequency_range.min = 0 ;
 		button_manual_start.set_text(to_string_short_freq(frequency_range.min));
-		frequency_range.max = stored_freq + 1000000;
+		if( stored_freq + 1000000 < MAX_UFREQ )
+			frequency_range.max = stored_freq + 1000000;
```

Comment:
Maybe put 1000000 into a var as it's used quite a bit and would be very easy for a dev to accidentally miss a 0 on the end

Reply:
Suggestion implemented ;-)